packages/evolution-common/src/services/conditionals/__tests__/checkConditionals.test.ts

```diff
@@ -0,0 +1,429 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import each from 'jest-each';
+import { checkConditionals } from '../checkConditionals';
+
+const interview = {
+    responses: {
+        _activePersonId: 'bb6c33d8-5b68-4eb4-a031-2ffad0fd2ba5',
+        _isNull: null,
+        _isArray: ['a', 'b', 'c'],
+        _isEmptyArray: [],
+        _isString: 'a',
+        _isTrueBoolean: true,
+        _isNumber1: 1,
+        household: {
+            size: 1,
+            persons: {
+                'bb6c33d8-5b68-4eb4-a031-2ffad0fd2ba5': {
+                    age: 33
+                }
+            }
+        }
+    }
+};
+
+each([
+    [
+        'wrongPath === null, should return false.',
```

Comment:
Intéressant, est-ce que ça ne devrait pas être `true` ici? 'wrongPath === 1' `false`, mais avec `null`? Est-ce que le `null`est synonyme de `undefined`? ou on veut traiter `null` et `undefined` séparément pour les comparaisons?

Reply:
T'avais raison, j'avais oublié de mettre null comme ceci 'null' (Comme ce que fait les generateSurvey sur Excel présentement).  J'ai rajouté un test pour tester ceci et pour tester lorsque ­wrongPath === 'something', should return false.